src/components/NumberList/NumberList.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import './NumberList.css'
+import { contacts } from '../../constants/contacts';
```

Comment:
use direct path from the src folder if your application more than one folder up.

eg :  ` import {ModifyContact}  from 'components/NumberList/ModifyContact/ModifyContact'`

try to use default import . Try to use one file for only one component and export with default